_posts/2017-07-09-P3-Quality.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-07-09 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-07-09-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know what to focus on?
+
+Nowadays, it's still thought that quality is having automated functional tests. A manager worried about quality tends to hire QA-roles or to contract of an external company to automate functional tests.
```

Comment:
I would turn it differently:
It is still believed that quality in software is about automated functional tests.
`A manager worried about quality tends to hire QA-roles or to contract of an external company to automate functional tests.` -> ` tends to hire QAs`
`or to contract of an external company to automate functional tests.` -> `or to contract external companies to automate functional tests.`